src/test/e2e/api/admin/project/features.e2e.test.ts

```diff
@@ -3394,3 +3395,29 @@ test('should not be allowed to update with invalid strategy type name', async ()
         400,
     );
 });
+
+test('should list dependencies and children', async () => {
+    const parent = uuidv4();
+    const child = uuidv4();
+    await app.createFeature(parent, 'default');
+    await app.createFeature(child, 'default');
+    await app.addDependency(child, parent);
```

Comment:
added this new method since it may come in handy and also code is symmetrical when we do everything with app.doXYZ